mit_frontend/src/routers/ResetPassword.js

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
+import {Card, CardTitle} from 'material-ui/Card';
+import TextField from 'material-ui/TextField';
+import RaisedButton from 'material-ui/RaisedButton';
+import '../App.css';
+import styles from './styles';
+
+class ResetPassword extends Component {
```

Comment:
I know you are only working on UI which might not include data model and interaction with DB. Where in the code will be the place to call Joseph's code to update password of the user account in DB?